cognizant-studio-academy/Inventory-Management
# F/166152373

#### What's this PR do?
This PR contains Welcome page with manage Inventories menu if the logged in user id is 1
#### Where should the reviewer start?
All Files in Client/ and Server/ folders
#### How should this be manually tested?
Step 1 : Login
Step 2: check the page whether the manage Inventories menu is displaying or not
#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/166152378
